cmd/cody-gateway/internal/httpapi/completions/flagging.go

```diff
@@ -22,7 +23,25 @@ type flaggingConfig struct {
 	PromptTokenBlockingLimit       int
 	MaxTokensToSampleFlaggingLimit int
 	ResponseTokenBlockingLimit     int
+
+	// If false, the flaggingResult returned from isFlaggedRequest will never have shouldBlock set to true.
```

Comment:
Nit: I find this hard to read. How about
```suggestion
	// If false, flaggingResult.shouldBlock will always be false when returned by isFlaggedRequest.
```

Reply:
Yup, definitely a clear improvement.